Uchu.World/Client/CdClient/CacheTable/EntryCacheTable.cs

```diff
@@ -53,7 +53,7 @@ public EntryCacheTable(Type type, PropertyInfo index) : base(type, index)
             index = ConvertKey(index);
             
             // Populate the cache entry.
-            await this._semaphore.WaitAsync();
+            await _semaphore.WaitAsync();
```

Comment:
is it intentional that the `this.` before `_semaphore` is removed?

Reply:
`_semaphore` is now static. The `CdClientContext` was static before, which led to a concurrency exception in rare cases.

Reply:
Ah, didn't notice that, makes sense :+1: